assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { dataFontEm } from '../../../../units';
+
+const DEFAULT_FONT = {
+  id: 'default-text',
+  content: __('Fill in some text', 'web-stories'),
+  fontSize: dataFontEm(1.1),
+  fontWeight: 400,
+  font: {
```

Comment:
There's already `TEXT_ELEMENT_DEFAULT_FONT` in `assets/src/edit-story/app/font/defaultFonts.js`  which we can use.

Reply:
Do you mean we should import the TEXT_ELEMENT_DEFAULT_FONT and use it in textPresets.js or to move the DEFAULT_FONT in this file to the defaultFonts.js

Reply:
TEXT_ELEMENT_DEFAUL_FONT seems to have a series of different weights instead of just one weight? I'm a bit confused about how the TEXT_ELEMENT_DEFAULT_FONT can be used in the same way.

Reply:
Not sure why this got resolved. But IMHO this is a critical part to address. Yes, a standard font definition should have all the weights/styles/variants that it supports so that we do not have to query any database when processing it.

Reply:
To clarify, we need to import `TEXT_ELEMENT_DEFAULT_FONT` from the `defaultFonts.js` and use it everywhere here.

Reply:
PTAL. Was the new change what you meant?